package.json

```diff
@@ -57,25 +57,24 @@
     "@types/configstore": "^4.0.0",
     "@types/inquirer": "^6.5.0",
     "async-mutex": "^0.2.4",
+    "bluebird": "^3.7.2",
```

Comment:
Do we really need to use bluebird? Can we just use native promises and native async await?

Reply:
yes we can definitely do that. Not sure why this is in there tbh.

Reply:
also removed some other unnecessary deps: `readline`, `core-js`, `inquirer`, `pem` and `cli-progress`